src/python/pants_release/git.py

```diff
@@ -73,11 +73,8 @@ def github_repo() -> github.Repository.Repository:
 
     try:
         gh = github.Github(auth=github.Auth.Token(token))
-        user = gh.get_user()
```

Comment:
This should be available via `GITHUB_ACTOR` within GItHub Actions, right?

Reply:
https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

Reply:
Yes, but this is also meant to be run locally as well. :man_shrugging: Trying to find out the right code that'll work in all cases just didn't seem worth it for the benefit of a little message.

Reply:
Maybe use `GITHUB_TOKEN` from the environment if it exists? The comment suggests that `gh` looks for `GH_TOKEN` which is not `GITHUB_TOKEN`.

Reply:
It uses both

Reply:
```console
josh@cephandrius:~/work/pants$ gh help environment | grep GH_TOKEN
GH_TOKEN, GITHUB_TOKEN (in order of precedence): an authentication token for github.com
```

Reply:
What about checking `GITHUB_ACTOR` first and then continuing to use the token if that's not set?